canonical-web-and-design/anbox-cloud.io
# Creates Telco vertical page on Anbox

## Done

Created Telco vertical page on Anbox site

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/telco
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy doc](https://docs.google.com/document/d/1DEG0SELYXIJCdqwaJOWM6R1yCiTnEub4kaK0aX4bT2I/edit)
- Compare against Zeplin [design](https://app.zeplin.io/project/60814423dc48d222d04433eb/screen/6082d1bf760f5411766bd57f)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4008
